js/viewer/annotation/annotation-group.js

```diff
@@ -250,12 +315,42 @@ function AnnotationGroup(id, anatomy, description, parent){
     * This function changes the color(stroke) of the each annotation to the new value passed as param
```

Comment:
This jsdoc was supposed to be for the `updateStroke` below, so please move it. Also add jsdoc for the new `changeArrowStroke` function.

Reply:
Done, updated the comments.